### ISSUE TITLE
Getting balances of parent and empty/zero accounts

### ISSUE THREAD

**Issue by the0** [ledger/ledger#560]
Hi there,

I recently encountered with behavior discussed here:

https://groups.google.com/forum/#!topic/ledger-cli/86QkKpMrluU

As far as I understand, it was considered a bug. Are there plans to fix it?

I can confirm this behavior for the latest (gd7bc4bab) commit.

**Comment by Fjan** [comment]
I ran into this as well. I would say that the behaviour of not showing empty parent accounts is design decision that's debatable, but not showing non-empty parent accounts if they've not been used is more problematic.

So you can get this in a balance:
```
Financial:Expenses:Other  80 USD
```

But you do not get any mention of `Financial:Expenses` if it hasn't been used in the period.
 The --empty option would not be a good name for an option to show this, since it's not empty, perhaps --intermediate?

**Comment by jwiegley** [comment]
I don't know if I agree with this anymore, since the point of the display is to be as compact as possible. If this behavior is needed, I think it should be under an option. I personally preference to see `Assets:Checking` if that's the only line that will appear in the output, since splitting it to two lines now still doesn't communicate to me which "posted to but now zero" siblings are being implied by the report.

**Comment by Fjan** [comment]
I agree, the --empty option already suffices for me. As an avid ledger user I'm now completely used to the current behavior so no need to change anything on my account. But I remember being puzzled the first time I encountered a missing account, before realizing only one sibling was non-zero, so the parent was elided.
